### DISCUSSION TITLE
Untyped property in PayerCost interface

### DISCUSSION THREAD

**Discussion by pedropereiradev** [mercadopago/sdk-react#54] [Q&A] [answered]
I was trying to use the `recommended message` property to show installments when using `getInstallments` function and it is untyped. It should be nice if you add it in next releses.

**Comment by meliguicarvalho** [comment, marked as answer]
Hello! Thank you for your contribution.

We have added to our backlog a task to add this typing to the SDK. 
When we have news I will let you know.

**Comment by ghost** [comment]
Fixed in https://github.com/mercadopago/sdk-react/pull/63
